src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -151,20 +151,20 @@ private ImmutableArray<TypeSymbol> GetDeclaredConstraintTypes()
             }
 
             var moduleSymbol = containingType.ContainingPEModule;
-
-            EntityHandle[] constraints;
+            var metadataReader = moduleSymbol.Module.MetadataReader;
+            GenericParameterConstraintHandleCollection constraints;
 
             try
             {
-                constraints = moduleSymbol.Module.GetGenericParamConstraintsOrThrow(_handle);
+                constraints = metadataReader.GetGenericParameter(_handle).GetConstraints();
```

Comment:
>constraints = metadataReader.GetGenericParameter(_handle).GetConstraints(); [](start = 16, length = 75)

It looks like we might need to extend try/catch because operations that used to be done by GetGenericParamConstraintsOrThrow are now done below.